### ISSUE TITLE
Mo2 not launching f4se steam deck

### ISSUE THREAD

**Issue by Aidanman2** [rockerbacon/modorganizer2-linux-installer#702]
clicking launch when the f4se shortcut is selected causes a split second black screen which will then default to the regular fallout.exe
changing the name of the f4se to falloutlauncher.exe opens script extender and the game correctly but not mo2 and no mods are loaded.

Any idea why this is happening?

Thanks!

**Comment by BigNervous138** [comment]
I’m having the exact same issue, have tried swapping names and adding a RunTime line in the f4se.ini but each method leads to more issues, I know it’s something to do with the SteamOS but have tried a variety of different Protons and still no luck

**Comment by Aidanman2** [comment]
Yeah I hope its an easy fix because I have done what you have as well with no luck.

**Comment by TheUncannyObserver** [comment]
I have this problem on desktop, launching the game from Steam goes launches F4SE and the game, MO2 never opens.
